juju-tutorials/how-to-fixing-attach-volume-to-a-k8s-instance.md

```diff
@@ -0,0 +1,336 @@
+[comment]: <> (How To Perform Fixing Attach Volume to Instance)
```

Comment:
Suggestion: How To Fix A Volume Attachment Failure Against An OpenStack Instance